reflex/components/radix/themes/components/radio_group.py

```diff
@@ -145,11 +145,7 @@ def create(
 
         # convert only non-strings to json(JSON.stringify) so quotes are not rendered
         # for string literal types.
-        if (
-            type(default_value) is str
-            or isinstance(default_value, Var)
-            and default_value._var_type is str
-        ):
+        if isinstance(default_value, (str, Var)) and default_value._var_type is str:
```

Comment:
I think this line change is not equivalent to the previous line

This seems more correct :
```suggestion
        if isinstance(default_value, str) or (isinstance(default_value, Var) and default_value._var_type is str):
```